tests/integration/integration_test.go

```diff
@@ -581,8 +581,11 @@ func TestDestroyStackRef_Cloud(t *testing.T) {
 	e := ptesting.NewEnvironment(t)
 	defer e.DeleteIfNotFailed()
 
-	output, _ := e.RunCommand("pulumi", "whoami")
-	organization := strings.TrimSpace(output)
+	defaultOrg, _ := e.RunCommand("pulumi", "org", "get-default")
```

Comment:
Hrm I'm not sure this is a good change for integration tests?  This means anyone logged in running integration tests will create their test stacks in the default org, but not sure they should be there?  Having them in a separate org seems to be the right thing to do to me?

Reply:
Noted! I remember that this was failing but I can dig a bit deeper. Since the changes here ended up being more invasive than I originally imagined, I'm planning on splitting the PR out across call sites; in the process I'll see if I can clean this up a bit!